modules/system/assets/ui/js/drag.scroll.js

```diff
@@ -399,7 +399,7 @@
     DragScroll.prototype.dispose = function() {
         this.scrollClassContainer = null
 
-        $(document).off('ready', this.proxy(this.fixScrollClasses))
```

Comment:
Is this the correct replacement? Note that this is using `.off` instead of `.on`. @w20k can you comment on this?

Reply:
@LukeTowers since you're disposing `DragScroll.prototype.dispose` you'll also want to unbind the event handlers that are attached to it.

Reply:
@LukeTowers , I guess it was a copy-past mistake :) @ayumihamsaki could you roll back '$(document).off('ready', fn);' From the 2.x version was removed ['on(ready, fn)'](https://jquery.com/upgrade-guide/3.0/#breaking-change-on-quot-ready-quot-fn-removed). Off + event still present.

Reply:
@LukeTowers Corrected resolve issues now for you to re-check.